client/sites-dashboard/components/sites-site-coming-soon.tsx

```diff
@@ -0,0 +1,78 @@
+import styled from '@emotion/styled';
+import { CSSProperties } from 'react';
+
+type Props = {
+	siteName?: string;
+	className?: string;
+	style?: CSSProperties;
+	lang: string;
+};
+
+const Root = styled.div( {
+	display: 'flex',
+	alignItems: 'center',
+	backgroundColor: '#117ac9',
+	borderRadius: 4,
+	boxSizing: 'border-box',
+} );
+
+const comingSoonTranslations: Record< string, string > = {
+	ar: 'قريبًا',
+	de: 'Demnächst verfügbar',
+	es: 'Próximamente',
+	fr: 'Bientôt disponible',
+	he: 'בקרוב',
+	id: 'Segera hadir',
+	it: 'Presto disponibile',
+	ja: '近日公開',
+	ko: '공개 예정',
+	nb: 'Kommer snart',
+	nl: 'Komt binnenkort',
+	pt: 'Em breve',
+	ro: 'În curând',
+	ru: 'Скоро будет',
+	skr: 'جلدی آندا پئے',
+	sv: 'Kommer snart',
+	tr: 'Pek yakında',
+	zh: '即将推出',
+};
+
+const getTranslation = ( lang?: string ) => {
+	let text = 'Coming soon';
```

Comment:
The coming soon notice on the front end has a capital "S".

Doing this would mean we callback to the user's interface language, which is more likely to match the site language than English IMO.
```suggestion
	let text = __( 'Coming Soon' );
```

Reply:
That's interesting - the other translations are not capitalised and we don't use `text-transform: capitalized` either. But I will make it match. Thanks.